File: News_Service_JDBC/src/main/java/com/dunice/projectNews/mapper/TagMapper.java
package com.dunice.projectNews.mapper;

import com.dunice.projectNews.dto.response.TagResponse;
import com.dunice.projectNews.models.Tag;
import org.mapstruct.Mapper;
import org.mapstruct.MappingConstants;

@Mapper(componentModel = MappingConstants.ComponentModel.SPRING)
public interface TagMapper {
    TagResponse tagToTagDto(Tag tag);
}

File: News_Service_Kotlin/src/main/kotlin/com/dunice/advancedServiceKotlin/mapper/TagMapper.kt
package com.dunice.advancedServiceKotlin.mapper

import com.dunice.advancedServiceKotlin.dto.response.TagResponse
import com.dunice.advancedServiceKotlin.models.Tag
import org.mapstruct.Mapper
import org.mapstruct.MappingConstants

@Mapper(componentModel = MappingConstants.ComponentModel.SPRING)
interface TagMapper {
    fun tagToTagDto(tag: Tag): TagResponse
}
